scripts/travis-run

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -e  # Exit on non-zero status
+# set -x  # Print commands run
+# set -u  # Treat unset variables as an error
+
+
+echo "Here in travis-run..."
+echo $BUILD_DOCS
+echo $BUILD_DOCKER
+echo $JSLINT
+
+if [ ${JSLINT:-0} -ne 0 ]
+then
+    npm install
+    make jslint
+fi
+
+
+if [ ${BUILD_DOCS:-0} -ne 0 ]
+then
+    pip install -r requirements/docs.txt
+    sphinx-build -b html -d doctrees docs html
+fi
+
+if [ ${BUILD_DOCKER:-0} -ne 0 ]
+then
+
+    if [ -x $(command -v docker-compose) ]
+    then
+        echo "Overwriting existing docker-compose."
+        docker-compose -v
+    else
+        echo "Installing docker-compose ${DOCKER_COMPOSE_VERSION}."
+    fi
+    mkdir -p downloads
+    DOCKER_COMPOSE_FILE=downloads/docker-compose-${DOCKER_COMPOSE_VERSION}
+    if [ ! -f $DOCKER_COMPOSE_FILE ]
+    then
+        wget -q --waitretry=1 --retry-connrefused -T 10 \
+            -O downloads/docker-compose-${DOCKER_COMPOSE_VERSION} \
+            https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m`
+    fi
+    chmod +x $DOCKER_COMPOSE_FILE
+    sudo cp $DOCKER_COMPOSE_FILE /usr/local/bin/docker-compose
+
+    # XXX would be nice to have code comments explaining each of these
+    VERSION=latest make build-base
+    VERSION=latest make build-kuma
+
+    docker-compose --version
+    docker-compose up -d
+    # Hang tight until MySQL appears to be working
+    docker-compose exec -T web urlwait mysql://root:kuma@mysql:3306/developer_mozilla_org 30
+    # Hang tight until Elasticsearch appears to be working
+    docker-compose exec -T web urlwait http://elasticsearch:9200 30
+
+    # XXX Would it help to bundle all of these make commands into one?!
+    docker-compose exec -T web make webpack
```

Comment:
`make build-static` includes the `make webpack, so I don't think we need it here, or perhaps it needs to be run prior to the `make localecompile`? I don't think so, but maybe I'm missing something.

Reply:
First of all, my bad for explicitly running `make webpack` when it's part of `build-static`. 

Right now, the problem I have is quite luxurious. The tests passed but I noticed that the super-expensive `make build-static` was called twice. First by `kuma/Dockerfile` and then by this `docker-compose ...` command.